### PR TITLE
Add autotest/bundler tip to README

### PR DESCRIPTION
**Pull request by vandrijevik**
This is my attempt at including a tip for the gotcha I encountered when using `autotest/bundler` + `minitest-autotest`, as encouraged [in this comment](https://github.com/seattlerb/minitest-autotest/issues/10#issuecomment-165042164).

I think that the tips section of the README is the most appropriate place, and hope that my language is not too verbose!

### CODE REVIEW

**Comment by vandrijevik** [comment]
Hi @zenspider, it occurs to me that I never mentioned you in the body of this PR, so you may not have gotten a notification about it. Does this look okay?

File: README.rdoc [hunk 1]
```diff
@@ -78,6 +78,14 @@ Then add the following to the ".autotest" file:
 
     require "autotest/spec"
 
+If you use autotest/bundler, make sure your Gemfile includes
+the minitest-autotest gem. That way, the minitest-server plugin
+(which is used by minitest-autotest) will be put on the load path
+by bundler, and will be loaded by minitest. Otherwise, you may
+get the following error message:
```
Comment (zenspider) [inline README.rdoc]: Technically... yes, but the real tip is that they should be putting minitest-autotest in their Gemfile. It already declares a dependency on minitest-server here : https://github.com/seattlerb/minitest-autotest/blob/master/Rakefile#L14

Comment (vandrijevik) [inline README.rdoc]: You’re right! I simplified the language in a432b3b to not distract from the real tip. Do you think the bit about the error message is okay (so someone could search and find the tip if they encounter it), or should I get rid of that too?

**Comment by zenspider** [comment]
Merged! Thanks!

**Pull request closed without merging**